node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/editor/TextBufferImpl.java

```diff
@@ -96,9 +97,12 @@ public void delete(Position start, Position end) {
     }
 
     private void deleteLines(int startIndex, int endIndex) {
-        for (int i = startIndex; i < endIndex; ++i) {
-            lines.remove(startIndex);
+        int fromIndex = endIndex;
+        for (int toIndex = startIndex; fromIndex <= getMaxLineIndex(); ++toIndex, ++fromIndex) {
+            lines.set(toIndex, lines.get(fromIndex));
```

Comment:
This is neater than the original, anyway :)
Mini-nit: I'd declare `fromIndex` in the `for` construct as well, for maximum uniformity!

Reply:
Oh, this _is_ the original 😝

Reply:
It's just that it now fits the list type 😁

Reply:
exactly